docusaurus/docs-internals/editor-controls/data-controls/toggle.md

```diff
@@ -131,19 +194,156 @@ const getToolbarContols = ({ getValue }) => {
       type: "select",
       choices: [
         { title: "Youtube", value: "youtube" },
-        { title: "Custom", value: "custom" },
-      ],
+        { title: "Custom", value: "custom" }
+      ]
     },
     {
-      id: "size",
+      id: "align",
       type: "toggle",
       disabled: videoType === "custom",
       choices: [
         { icon: "nc-text-align-left", title: "Left", value: "left" },
         { icon: "nc-text-align-center", title: "Center", value: "center" },
-        { icon: "nc-text-align-right", title: "Right", value: "right" },
-      ],
-    },
+        { icon: "nc-text-align-right", title: "Right", value: "right" }
+      ]
+    }
   ];
 };
 ```
+
+#### Display example
+In this example, with `display: "block"`, the label will be rendered on the first row and the control on the second.
+
+```js
+{
+  id: "align",
+  type: "toggle",
+  display: "block"
+}
+```
+
+#### Helper examples
+The helper object contains a content property with the value `"help text"`, which will be displayed as additional
+guidance or information for the user.
+
+```js
+{
+  id: "align",
+  type: "toggle",
+  helper: {
+    content: "help text"
+  }
+}
+```
+
+When the helper object contains a position property with the value `"top-start"`, it indicates that the helper text will
+be displayed at the top left corner of the icon.
+
+```js
+{
+  id: "align",
+  type: "toggle",
+  helper: {
+    content: "help text",
+    position: "top-start"
+  }
+}
+```
+
+#### Default value examples
+
+In this example, the toggle control that has the value `"center"` by default.
+
+```js
+{
+  id: "align",
+  type: "toggle",
+  default: {
+    value: "center"
+  }
+}
+```
+
+#### CSS examples
+
+Change the color of the `.brz-toggle` element with CSS using a toggle control value. In this example, if value is `"left"`, set the color to red else set the color to blue.
+```js
+{
+  id: "align",
+  type: "toggle",
+  style: ({ value }) => {
+    if (value === "left") {
+      return {
+        "{{WRAPPER}} .brz-toggle": {
+          color: "red"
+        }
+      }
+    }
+
+    return {
+      "{{WRAPPER}} .brz-toggle": {
+        color: "blue"
+      }
+    }
+  }
```

Comment:
better make an example with aligns:
```ts
if (value.value === "left"){
  return {
        "{{WRAPPER}} .brz-toggle": {
          "justify-content": "start";
        }
      }
// ...
}
```

and do the same checks for `center` and `right`